enhancements/windows-containers/windows-machine-config-operator.md

```diff
@@ -96,15 +96,15 @@ will have all the pre-requisite steps needed for setting the VM up for key based
 SSH connections.
 
 When the cluster admin wants to bring up Windows worker nodes, they will create
-a [MachineSet](https://github.com/openshift/machine-api-operator/blob/master/config/machineset.yaml)
-where they reference a Windows OS image which has the Docker container runtime
-add-on enabled. They will also specify the user data secret created by WMCO as
+a [MachineSet](https://github.com/openshift/machine-api-operator/blob/master/config/machineset.yaml).
+They will also specify the user data secret created by WMCO as
 described above. WMCO will watch for Machine objects with a special label that
 indicate that they map to Windows VMs. Once the Machine object is in the
 ["provisioned" phase](https://github.com/openshift/enhancements/blob/master/enhancements/machine-api/machine-instance-lifecycle.md),
 WMCO will connect to the associated VM to transfer the binaries and perform the
 following steps for it to join the cluster as a worker node:
 * Transfer the required binaries
+* Configure containerd as the default container runtime
```

Comment:
Again, please mention the WMCO version where this will happen. Example: WMCO 6.0.0+ etc.